src/util-decode-der-get.c

```diff
@@ -72,6 +73,67 @@ static const char *Oid2ShortStr(const char *oid)
     return "unknown";
 }
 
+static time_t GentimeToTime(char *gentime, size_t length)
```

Comment:
Seems all callers pass valid c strings, and pass the result of strlen() as length. That means we can ditch the argument and call strlen in the function itself